src/main/java/edu/kis/vh/nursery/DefaultCountingOutRyhmer.java

```diff
@@ -0,0 +1,47 @@
+package edu.kis.vh.nursery;
+
+/**
+ * @author student
+ *
+ */
+public class DefaultCountingOutRyhmer {
+
+	private static final int _NUMBERS_SIZE = 12;
+	private static final int _1 = -1;
+	private final int[] NUMBERS = new int[_NUMBERS_SIZE];
+	private int total = _1;
+
+	 public int getTotal() {
+		return total;
+	}
+
+	public void setTotal(int total) {
+		this.total = total;
+	}
+
+	void countIn(int in) {
+		if (!isFull())
+			NUMBERS[++total] = in;
+	}
+
+	 boolean callCheck() {
+		return total == -1;
+	}
+
+	 boolean isFull() {
+		return total == 11;
```

Comment:
Another constant. It is related to _NUMBERS_SIZE (yet, the name is misinforming)